pages/learn/develop/multicontainer.md

```diff
@@ -66,7 +66,7 @@ restart: always
 
 Setting `network_mode` to `host` allows the container to share the same network namespace as the host OS. When this is set, any ports exposed on the container will be exposed locally on the device. This is necessary for features such as bluetooth.
 
-To store data in [persistent storage][persistent-storage], you'll want to make sure to use the `volumes` field to link a directory in your container to the `resin-data` volume. Your named volume should be specified at the top-level of the `docker-compose.yml`, with the container path defined in the service:
+To store data in [persistent storage][persistent-storage], use the `volumes` field of the service to link the `/data` directory in your container to the `resin-data` named volume. The `resin-data` named volume should also be specified at the top-level of the `docker-compose.yml`:
```

Comment:
It would be good to explain that arbitrary named volumes can be created. Also, note that tge volumes will be persisted as long as there’s a volume with that name in every release of the application, so if you change the name of a volume, the data will be lost.